src/app/components/modules/module-config.ts

```diff
@@ -89,6 +90,13 @@ export const config: Config = {
 		defaultHeight: 1,
 		defaultWidth: 2
 	},
+	simplifiedLunch: {
+		displayName: 'Simplified Lunch',
```

Comment:
I like that the component is named `SimplifiedLunchComponent` because it may conflict with the main `LunchComponent`, however, we may want the display name to be simply "Lunch" because it saves space and there isn't any other module which may lead to confusion. The `SimplifiedSchedule` module is named the way it is because there's another `Schedule` module, and the `SimplifiedSchedule` is literally a less-detailed version of the former.